spec/compiler/crystal/tools/implementations_spec.cr

```diff
@@ -368,4 +368,104 @@ describe "implementations" do
     Bar.bar
     )
   end
+
+  it "find class implementation" do
+    assert_implementations %(
+    ༓class Foo
+    end
+
+    F‸oo
```

Comment:
There are some weird chars before `class` and just after the `F` of `Foo` on L377 (here and on all other specs)

Reply:
That's intended (see whole spec for details).

Reply:
Oooh right, that's neat actually :smiley: